app/src/main/java/com/github/multimatum_team/multimatum/activity/MainSettingsActivity.kt

```diff
@@ -53,7 +73,7 @@ class MainSettingsActivity : AppCompatActivity() {
         val sensorPresent = sensorFound()
         // if no sensor available then button should be gray to show that it is disabled
         procrastinationDetectEnabledButton.alpha =
-            if (sensorPresent) BUTTON_ALPHA_NORMAL else BUTTON_ALPHA_DISABLED
+            if (sensorPresent) ALPHA_NORMAL else ALPHA_DISABLED
         if (!sensorPresent) {
```

Comment:
is these name are important ? could we have a more comprehensible name ? or it depends on some library ?

Reply:
These constants correspond to two values of the alpha component of the color of widgets. They are defined in the companion object of `MainSettingsActivity`. And I find them comprehensible, at least with the comment at line 74. Or do you have better names?